src/InputGroup/InputGroup.test.js

```diff
@@ -216,101 +140,69 @@ describe('<InputGroup />', () => {
 
     });
 
-    test('input text entered', () => {
-        const wrapper = mountComponentWithStyles(inputTextPosBefore);
-
-        // click up
-        wrapper.setState({ value: '' });
-        wrapper
-            .find('input[type="text"]')
-            .simulate('change', { target: { value: 'hello' } });
-
-        expect(wrapper.state('value')).toEqual('hello');
-    });
-
-    test('number input handle up and down', () => {
-        const wrapper = mountComponentWithStyles(numberInput);
-
-        // click up
-        wrapper.setState({ value: 0 });
-        wrapper
-            .find(
-                'button.fd-button--half.sap-icon--slim-arrow-up'
-            )
-            .simulate('click');
-        expect(wrapper.state('value')).toEqual(1);
-
-        wrapper
-            .find(
-                'button.fd-button--half.sap-icon--slim-arrow-down'
-            )
-            .simulate('click');
-        expect(wrapper.state('value')).toEqual(0);
-    });
-
-    describe('Prop spreading', () => {
-        test('should allow props to be spread to the InputGroup component (type text, addonPos after)', () => {
-            const element = mount(<InputGroup data-sample='Sample' />);
-
-            expect(
-                element.getDOMNode().attributes['data-sample'].value
-            ).toBe('Sample');
+    let setup = (props) => {
+        return mount(
+            <InputGroup {...props}>
+                <InputGroup.Addon>$</InputGroup.Addon>
+                <FormInput />
+            </InputGroup>);
+    };
+
+    describe('Default Rendering', () => {
+        let element = setup({
+            children: [<FormInput />, <InputGroup.Addon>@</InputGroup.Addon>]
```

Comment:
Is this necessary? Your setup on line 143 already has children in it. I would either edit the setup to not include any default children or edit your other tests to remove this and just call `let element = setup()`